src/utils/__tests__/ga-ecomm-helpers.test.js

```diff
@@ -0,0 +1,42 @@
+const { filterEcommEvents, isEcommEvent } = require('../');
+
+describe('Util: Filter Ecommerce Events', () => {
+  const [name, id, revenue] = ['fooname', 'fooid', 523.55];
+  const baseEvent = {
+    hitType: 'addTransaction',
+    customTrackerId: 'myTracker',
+    id,
+    name,
+    revenue,
+  };
+
+  it('should return an object with hitType and trackerId stripped out', () => {
+    const result = filterEcommEvents(baseEvent);
+    const expected = {
+      id,
+      name,
+      revenue,
+    };
+    expect(result).toEqual(expected);
+  });
+
+  it('should return true if event hitType is related to ecommerce', () => {
```

Comment:
These two tests for `isEcommEvent` are beautiful. It would be nice to have them wrapped in their own `describe` block though, just so its easier to distinguish them from the tests for `filterEcommEvents`.